jetpack.php

```diff
@@ -16,7 +16,7 @@
 
 define( 'JETPACK__MINIMUM_WP_VERSION', '5.5' );
 define( 'JETPACK__MINIMUM_PHP_VERSION', '5.6' );
-define( 'JETPACK__VERSION', '9.3-alpha' );
+define( 'JETPACK__VERSION', '9.3-alpha-20201221' );
```

Comment:
Was this for testing purposes or were you intending for this to go in?

Reply:
This was intentional in order to bust the caches, discussed here p1608286346170200-slack-CBG1CP4EN